meson.build

```diff
@@ -456,10 +457,10 @@ foreach lib:enabled_pmd_libs
 endforeach
 
 cndp_pmds_a_name = 'libcndp_pmds.a'
-run_command(mklib, cndp_pmds_a_name, libcndp_pmds_a, check: true)
+run_command(mklib, tempdir, cndp_pmds_a_name, libcndp_pmds_a, check: true)
 cndp_pmds_a = custom_target('libcndp_pmds_a_target',
     output: cndp_pmds_a_name,
-    input: join_paths('/tmp', cndp_pmds_a_name),
+    input: join_paths(tempdir, cndp_pmds_a_name),
     command:[find_program('cp'), '@INPUT@', cndp_pmds_a_name],
     install_dir: join_paths('lib', 'x86_64-linux-gnu'),
     install: true)
```

Comment:
If we add a run_command() after this custom_target() to remove the temp directory will that work?